### ISSUE TITLE
"ipfs add" feedback

### ISSUE THREAD

**Issue by slothbag** [ipfs/go-ipfs#2125]
I would love to get some more detailed feedback from the "ipfs add" command.

Things like:
* Number of files found and the related total number of chunks (about to be added)
* Count of skipped files (subset of above) because they already existed in IPFS datastore
* Count of files (subset of above) moved from cache to datastore 
* Count of chunks (and total MiB) skipped because they already existed in IPFS datastore
* Count of chunks (and total MiB) moved from cache to datastore
* Count of new files/chunks that actually had to be added to the datastore (total files - skipped)

With these stats I can witness IPFS start to de-duplicate my content over time. And I can optimize the way I structure my data so that I can let IPFS reuse chunked content more effectively.

Basically, I don't want to see IPFS say "added 1GiB", "added 1GiB".. etc.. I'd like to see "added 1GiB", "added 100MiB, skipped 900MiB already existing"

**Comment by slothbag** [comment]
Other higher priority issues than this
